requirements.txt

```diff
@@ -1,19 +1,19 @@
-jax >= 0.4.29, != 0.4.36, != 0.5.1, != 0.5.2, != 0.6.0, <= 0.6.2
+jax >= 0.5.0, != 0.5.1, != 0.5.2, != 0.6.0, !=0.7.1, < 0.8.0
```

Comment:
fwiw i think 0.7.1 should work, but for some reason gh installs an older version of equinox. But there were performance regressions in 0.7.0 and 0.7.1 so I think it's fine to exclude them